python/graphstorm/sagemaker/utils.py

```diff
@@ -243,6 +246,18 @@ def download_graph(graph_data_s3, graph_name, part_id, world_size,
 
     graph_data_s3 = graph_data_s3[:-1] if graph_data_s3.endswith('/') else graph_data_s3
 
+    # By default we assume the node mappings exist
+    # under the same path as the rest of the graph data
+    if not raw_node_mapping_prefix_s3:
+        raw_node_mapping_prefix_s3 = f"{graph_data_s3}/raw_id_mappings"
+    else:
+        raw_node_mapping_prefix_s3 = (
+            raw_node_mapping_prefix_s3[:-1] if raw_node_mapping_prefix_s3.endswith('/')
+            else raw_node_mapping_prefix_s3)
+        assert raw_node_mapping_prefix_s3.endswith("raw_id_mappings"), \
+            "node_mapping_prefix_s3 must end with 'raw_id_mappings'"
```

Comment:
@classicsong do you think this requirement is valid? I think I'd rather remove it to allow for more flexibility of how/where mappings are created.

Reply:
We can remove this assert, it is not necessary that the s3 prefix end with raw_id_mappings/